osu.Game.Rulesets.Mania/UI/Components/OddColumnType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Mania.UI.Components
+{
+    public class OddColumnType : ColumnType
```

Comment:
I don't like how this column type is inconsistent conpared to other types, how about abstracting the texture name properties and override here?